lib/manager/gradle/index.js

```diff
@@ -25,122 +25,53 @@ async function extractAllPackageFiles(config, packageFiles) {
     }
   }
 
-  await configureUseLatestVersionPlugin(config.localDir);
+  await updatesReport.createRenovateGradlePlugin(config.localDir);
   await executeGradle(config);
 
   gradle.init();
+
+  const dependencies = await updatesReport.extractDependenciesFromUpdatesReport(
+    config.localDir
+  );
+  if (dependencies.length === 0) {
+    return [];
+  }
+
   const gradleFiles = [];
   for (const packageFile of packageFiles) {
     const content = await platform.getFile(packageFile);
     if (content) {
-      const deps = await extractPackageFile(content, packageFile, config);
-      if (deps) {
-        gradleFiles.push({
-          packageFile,
-          manager: 'gradle',
-          ...deps,
-        });
-      }
+      gradleFiles.push({
+        packageFile,
+        manager: 'gradle',
+        datasource: 'maven',
```

Comment:
@rarkins I think last time I missed the datasource :?

Reply:
Yeah, but I think I fixed that the first time too, so it maybe doesn't explain the problem